InstallationGuides/GuidesCollection.md

```diff
@@ -4,8 +4,8 @@
 ## SM8150
 | Vendor  | Device Name | CodeName  | Guide Link                                                                |
 |---------|-------------|-----------|---------------------------------------------------------------------------|
-| Xiaomi  | K20 Pro     | Raphael   | [Github](https://github.com/graphiks/woa-raphael)                         |
-| Xiaomi  | Mi 9        | Cepheus   | [Github](https://github.com/woacepheus/Port-Windows-11-Xiaomi-Mi-9)       |
+| Xiaomi  | K20 Pro     | Raphael   | [Github](https://github.com/new-WoA-Raphael/woa-raphael)                         |
+| Xiaomi  | Mi 9        | Cepheus   | [Github](https://github.com/ivanvorvanin/Port-Windows-XiaoMI-9)       |
```

Comment:
should be https://github.com/ivnvrvnn/Port-Windows-XiaoMI-9

Reply:
Fixed I think